_posts/2015-04-17-Enums-match-mutation-and-moves.md

```diff
@@ -0,0 +1,673 @@
+---
+layout: post
+title: "Mixing matching, mutation, and moves in Rust"
+author: Felix S. Klock II
+description: "A tour of matching and enums in Rust."
+---
+
+One of the primary goals of the Rust project is to enable safe systems
+programming. Systems programming usually implies imperative
+programming, which in turns often implies side-effects, reasoning
+about shared, aliasable state, et cetera.
```

Comment:
You use the term "aliasable state" (and "unaliased" below), which readers may be unfamiliar with without a definition. I would probably go with a term like "exclusive" rather than "unaliased"; not quite sure what to replace "shared, aliasable state" with, perhaps "shared state with multiple live references" or something like that.